lang/rs/apis/cne/Cargo.toml

```diff
@@ -10,6 +10,7 @@ readme = "README.md"
 version = "0.1.0"
 edition = "2021"
 license = "BSD-3-Clause"
+keywords = ["networking","cloud native", "AF_XDP"]
```

Comment:
nit: any reason not to put a space after the first comma? It's just inconsistent. We might as well get it right the first time.

Reply:
Good catch. Cargo clippy didn't catch this :). I've updated PR